pkgs/top-level/ruby-packages.nix

```diff
@@ -1201,6 +1201,16 @@
     };
     version = "4.1.5";
   };
+  google-protobuf = {
```

Comment:
This file is autogenerated. 
To add a gem you need to add it to this [Gemfile](https://github.com/NixOS/nixpkgs/blob/4877a270df71049535d2b9bef112ade6d60d15bd/pkgs/development/ruby-modules/with-packages/Gemfile) and execute `./maintainers/scripts/update-ruby-packages`. 
See https://nixos.org/manual/nixpkgs/stable/#adding-a-gem-to-the-default-gemset for details